### PR TITLE
Minor changes to HTO

### PR DESCRIPTION
**Pull request by Camundaan**

### CODE REVIEW

**Comment by akeller** [comment]
@Camundaan please add me and @christinaausley as reviewers when you are ready for us to review these changes.

Also, we've changed how our documentation is [versioned](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#versions). You'll likely want to make your changes in both `/docs/` and `/versioned_docs/version-8.0/` if you want the changes in the current docs version (8.0) and future versions (Next and beyond).

**Comment by Camundaan** [comment]
Hi @akeller,
Thank you, completely forgot that I had to do that :)

Just to understand what I have to do: did I already make the changes in `/docs/` and I still have to change it in `/versioned_docs/version-8.0/`?

What is the easiest way to do that? Can I copy-paste the changes? Or do I have to create a new PR, edit the same changes in Visual Studio, then push again?

Sorry for having to deal with a Github noob here.

**Comment by Camundaan** [comment]
This is probably really easy, but I am struggling with how to do this.

**Comment by christinaausley** [comment]
No problem @Camundaan ! I totally get it. Take the following steps:
1. Open VSC as you would normally to make any docs changes.
2. Navigate to `>versioned_docs` > `>version-8.0` > `guides` > `getting-started-orchestrate-human-tasks.md`.
3. Make the same changes in this file as you made in the "next", regular, section of docs.
4. Save the file.
5. Ensure you copy any new screenshots referenced in the `versioned_docs` into the appropriate "versioned" `img` file.
6. Commit and push changes as usual.

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 1]
```diff
@@ -45,14 +47,11 @@ Variables are part of a process instance and represent the data of the instance.
 
 ### Implement a form
 
-1. To add a form and decide what's for dinner, return to the **Modeler** homepage and click **New > Form**.
+1. To add a form and decide what's for dinner, navigate to the **Modeler** Home page and click on the project you are working on. Then click **New > Form**.
```
Comment (christinaausley) [inline docs/guides/getting-started-orchestrate-human-tasks.md:50]: This should read "Navigate to the Modeler **Home** page and click the project you are working on. Then, click **New > Form**."

File: versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md [hunk 2]
```diff
@@ -45,14 +47,10 @@ Variables are part of a process instance and represent the data of the instance.
 
 ### Implement a form
 
-1. To add a form and decide what's for dinner, return to the **Modeler** homepage and click **New > Form**.
+1. To add a form and decide what's for dinner, navigate to the **Modeler** Home page and click on the project you are working on. Then click **New > Form**.
```
Comment (christinaausley) [inline versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md:50]: Same comment here.

**Review by christinaausley — CHANGES_REQUESTED** [review]
Left two comments to adjust the text -- once you've done this, I'll approve to merge!

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 3]
```diff
@@ -30,12 +30,14 @@ To create an automated process with [user tasks](/components/modeler/bpmn/user-t
...
-   ![wrench icon example](./img/wrench-icon-example.png)
+   ![wrench icon example](./img/wrench-icon-example-2.png)
 6. Add a descriptive name using the properties panel. In this case, we've named ours `Decide what's for dinner`.
 7. Append an exclusive [gateway](/components/modeler/bpmn/gateways.md) to your user task by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 8. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
 9. Sequence flows require expressions to access variables from the form we'll create below to determine what to eat for dinner. To add an [expression](/components/concepts/expressions.md), click on the sequence flow to view the properties panel, and open the **Condition** tab to insert a conditional expression.
-10. Attach an end event to the two user tasks.
+10. Our sequence flow will have the expressions of `= Meal = "Salad"`on one side and `= Meal = "Chicken"`on the other.
```
Comment (akeller) [inline docs/guides/getting-started-orchestrate-human-tasks.md:38]: This is a bit of a nitpick. Steps in a guide should be action-oriented. Why not include this with step 9?

**Review by akeller — COMMENTED** [review]
We now have two image files - `wrench-icon-example` and `wrench-icon-example-2`. Are you still using `wrench-icon-example`? Or can we delete it and rename `wrench-icon-example-2`?

File: docs/guides/getting-started-orchestrate-human-tasks.md (hunk 3 above)
Comment (akeller) [reply docs/guides/getting-started-orchestrate-human-tasks.md:38]: Or perhaps re-write this to be an action - "Verify the sequences flows have the following expressions"

Comment (christinaausley) [reply docs/guides/getting-started-orchestrate-human-tasks.md:38]: Agreed here.

**Comment by christinaausley** [comment]
Hi @Camundaan -- this build is failing because you refer to `versioned_docs/version-8.0/guides/img/wrench-icon-example-2.png` when this image filename does not exist. You'll need to update this image reference to the exact name of the png file you want to use.
Screen Shot 2022-08-19 at 9 18 28 AM

File: docs/guides/getting-started-orchestrate-human-tasks.md [hunk 4]
```diff
@@ -35,7 +35,9 @@ To create an automated process with [user tasks](/components/modeler/bpmn/user-t
 7. Append an exclusive [gateway](/components/modeler/bpmn/gateways.md) to your user task by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 8. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
 9. Sequence flows require expressions to access variables from the form we'll create below to determine what to eat for dinner. To add an [expression](/components/concepts/expressions.md), click on the sequence flow to view the properties panel, and open the **Condition** tab to insert a conditional expression.
-10. Attach an end event to the two user tasks.
+10. Verify the sequences flows have the following expressions `= Meal = "Salad"`on one side and `= Meal = "Chicken"`on the other.
```
Comment (christinaausley) [inline docs/guides/getting-started-orchestrate-human-tasks.md:38]: Please add the following punctuation for completeness:
10. Verify the sequences flows have the following expressions**:** `= Meal = "Salad"`on one side**,** and `= Meal = "Chicken"`on the other.

File: versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md [hunk 5]
```diff
@@ -35,7 +35,9 @@ To create an automated process with [user tasks](/components/modeler/bpmn/user-t
 7. Append an exclusive [gateway](/components/modeler/bpmn/gateways.md) to your user task by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 8. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
 9. Sequence flows require expressions to access variables from the form we'll create below to determine what to eat for dinner. To add an [expression](/components/concepts/expressions.md), click on the sequence flow to view the properties panel, and open the **Condition** tab to insert a conditional expression.
-10. Attach an end event to the two user tasks.
+10. Our sequence flow will have the expressions of `= Meal = "Salad"`on one side and `= Meal = "Chicken"`on the other.
```
Comment (christinaausley) [inline versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md:38]: Please adjust this doc as @akeller mentioned to ensure it is an action item. Note that I adjusted the punctuation of how you phrased this in the original doc, so I would advise just copying/pasting here.

**Review by christinaausley — CHANGES_REQUESTED** [review]
Please make the same change to Step 10 you made in the `next` docs to the versioned 8.0 doc.

**Review by christinaausley — APPROVED** [review]

**Review by akeller — APPROVED** [review]

**Pull request merged**